ngraph/core/include/ngraph/op/scatter_update.hpp

```diff
@@ -16,10 +16,7 @@ namespace v3 {
 ///
 class NGRAPH_API ScatterUpdate : public util::ScatterBase {
 public:
-    static constexpr NodeTypeInfo type_info{"ScatterUpdate", 3};
-    const NodeTypeInfo& get_type_info() const override {
-        return type_info;
-    }
+    NGRAPH_RTTI_DECLARATION;
```

Comment:
Please resolve merge conflicts, these operations were moved to openvino folder and already use OPENVINO_RTTI macros